### ISSUE TITLE
Can import rpy2, but can't import importr function.

### ISSUE THREAD

**Issue by happypycode** [rpy2/rpy2#686]
Raise:
```python
>>> import rpy2
>>> from rpy2.robjects.packages import importr
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\ProgramData\Anaconda3\lib\site-packages\rpy2\robjects\__init__.py", line 19, in <module>
    from rpy2.robjects.robject import RObjectMixin, RObject
  File "C:\ProgramData\Anaconda3\lib\site-packages\rpy2\robjects\robject.py", line 10, in <module>
    rpy2.rinterface.initr_simple()
  File "C:\ProgramData\Anaconda3\lib\site-packages\rpy2\rinterface.py", line 796, in initr_simple
    status = embedded._initr()
  File "C:\ProgramData\Anaconda3\lib\site-packages\rpy2\rinterface_lib\embedded_mswin.py", line 22, in _initr_win32
    rstart.ReadConsole = callbacks._consoleread
TypeError: initializer for ctype 'int(*)(char *, char *, int, int)' must be a pointer to same type, not cdata 'int(*)(char *, unsigned char *, int, int)'
```

**Comment by lgautier** [comment]
Duplicate of issue #682 . This was fixed in PR #685 .

**Comment by happypycode** [comment]
> Duplicate of issue #682 . This was fixed in PR #685 .

Thank you very much, I build it from source not install from pip, this issue was fixed. using pip still have this issue.
